src/App.tsx

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
```

Comment:
a bit clearer this way
```suggestion
  const [pressedKey, setPressedKey] = useState('');
```